vaadin-confirm-dialog-flow-parent/vaadin-confirm-dialog-flow-integration-tests/src/test/java/com/vaadin/flow/component/confirmdialog/tests/FeaturesDiyIT.java

```diff
@@ -15,85 +15,136 @@
  */
 package com.vaadin.flow.component.confirmdialog.tests;
 
+import java.util.Optional;
+
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 import org.openqa.selenium.By;
 
-import com.vaadin.flow.component.confirmdialog.tests.helpers.Actionwords;
-import com.vaadin.tests.AbstractParallelTest;
+import com.vaadin.flow.component.button.testbench.ButtonElement;
+import com.vaadin.flow.component.confirmdialog.ConfirmDialog;
+import com.vaadin.flow.component.confirmdialog.testbench.ConfirmDialogElement;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.testbench.ElementQuery;
+import com.vaadin.testbench.TestBenchElement;
+import com.vaadin.tests.AbstractComponentIT;
 
-public class FeaturesDiyIT extends AbstractParallelTest {
+@TestPath("vaadin-confirm-dialog/FeaturesDiy")
+public class FeaturesDiyIT extends AbstractComponentIT {
 
-    public Actionwords actionwords = new Actionwords(this, FeaturesDiy.class);
+    @Before
+    public void init() {
+        open();
+    }
 
     @Test
     public void testReminderDialog() throws Exception {
-        String sampleName = "SampleConfirmDialog";
-        actionwords.iHaveSampleDialog(sampleName);
-        actionwords.iOpenDialogDialog(sampleName);
-        actionwords.iCompareTheDialogToReferenceImage(sampleName);
```

Comment:
We have disabled screenshot comparison long ago when moving to monorepo.
I'll delete the reference screenshots folder for `ConfirmDialog` in a separate PR.